installers/helm/postgres-operator/values.yaml

```diff
@@ -69,7 +69,8 @@ pgo_apiserver_port: "8443"
 pgo_apiserver_url: "https://postgres-operator"
 pgo_client_cert_secret: "pgo.tls"
 pgo_client_container_install: "false"
-pgo_client_version: "4.4.0"
+pgo_client_install: "true"
+pgo_client_version: "v4.4.0"
```

Comment:
I think it'd be good if we can leave the `v` off of this, and handle that for the user. and/or be able to accept `4.4.0` and `v4.4.0`

Reply:
Sure. This will require a minor change to the ansible